create_database.sh

```diff
@@ -46,14 +46,16 @@ SQL_CREATE_TABLE_SAVED_CONTENT="CREATE TABLE saved_content (
 
 SQL_CREATE_TABLE_CHAT_ROOMS="CREATE TABLE chat_room (
   id UUID DEFAULT gen_random_uuid() PRIMARY KEY,
-  name VARCHAR(100) NOT NULL,
-  created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+  chat_name VARCHAR(255) NOT NULL,
+  time_created TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
 );"
-SQL_CREATE_CHAT_ROOM_PARTICIPANT="CREATE TABLE chat_room_participant (
+
+SQL_CREATE_TABLE_USER_CHATROOM_XREF="CREATE TABLE user_chatroom_xref (
```

Comment:
because you changed this variable name, you have to update that below (line 80) to initialize the new table into the DB